sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/pipeline/TelemetryItemExporter.java

```diff
@@ -154,6 +161,21 @@ CompletableResultCode internalSendByBatch(TelemetryItemBatchKey telemetryItemBat
         return telemetryPipeline.send(byteBuffers, telemetryItemBatchKey.connectionString, listener);
     }
 
+    private boolean isMicrometerJvmMetric(TelemetryItem item) {
+        boolean result = false;
+        MonitorDomain md = item.getData().getBaseData();
+        if (md instanceof MetricsData) {
+            MetricsData metricsData = (MetricsData) md;
+            if (!metricsData.getMetrics().isEmpty()) {
+                MetricDataPoint point = metricsData.getMetrics().get(0);
+                String metricName = point.getName();
+                return metricName.startsWith("jvm_");
```

Comment:
Should I create a specific list of micrometer metric names to exclude or is this fine?